src/databricks/labs/ucx/hive_metastore/table_migration_status.py

```diff
@@ -108,15 +108,19 @@ def get_seen_tables(self) -> dict[str, str]:
         return seen_tables
 
     def is_migrated(self, schema: str, table: str) -> bool:
-        results = self._backend.fetch(
-            f"SHOW TBLPROPERTIES {escape_sql_identifier(schema + '.' + table)} ('upgraded_to')"
-        )
-        for result in results:
-            if "does not have property" in result.value:
-                continue
-            logger.info(f"{schema}.{table} is set as migrated")
-            return True
-        logger.info(f"{schema}.{table} is set as not migrated")
+        try:
+            results = self._backend.fetch(
+                f"SHOW TBLPROPERTIES {escape_sql_identifier(schema + '.' + table)} ('upgraded_to')"
+            )
+            for result in results:
+                if "does not have property" in result.value:
+                    continue
+                logger.info(f"{schema}.{table} is set as migrated")
+                return True
+            logger.info(f"{schema}.{table} is set as not migrated")
+        except NotFound:
+            # If the source table doesn't exist, it will not be shown as migrated
+            logger.info(f"{schema}.{table} set as a source does no longer exist")
```

Comment:
```suggestion
            logger.warning(f"failed-to-migrate: {schema}.{table} set as a source does no longer exist")
            return True
```
1) log warning, so it gets to dashboard
2) if we don't return true, there'll be an attempt to migrate table, which will also fail, because there's nothing to migrate from